cmake/crypto.cmake

```diff
@@ -18,7 +18,7 @@ file(GLOB_RECURSE EVERCRYPT_SRC "${EVERCRYPT_PREFIX}/*.[cS]")
 
 if("sgx" IN_LIST COMPILE_TARGETS)
   add_library(
-    evercrypt.enclave STATIC ${EVERCRYPT_SRC} ${CCF_DIR}/3rdparty/stub/stub.c
+    evercrypt.enclave STATIC ${EVERCRYPT_SRC} ${CCF_DIR}/3rdparty/stub/mem.c
```

Comment:
Looks like this file isn't included in the PR? `stub.c` removed, `time.c` added, but no `mem.c`

Reply:
Weird, I could have sworn I'd git mv'ed stub.c mem.c, and that's why I added only the other one. Oh well. Thanks!